compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -291,6 +300,16 @@ class ROCMTargetBackend final : public TargetBackend {
     pb.registerLoopAnalyses(lam);
     pb.crossRegisterProxies(lam, fam, cgam, mam);
 
+    for (std::string pluginFN : passPlugins) {
```

Comment:
This will cause the strings to be copied. This should use `StringRef` or `const std::string&`. Also, it's not clear to me what `FN` stands for.